### DISCUSSION TITLE
no output after hours of work?

### DISCUSSION THREAD

**Discussion by Schlicki** [TheLastGimbus/GooglePhotosTakeoutHelper#317] [I NEED HELP]
Hi,
I have run the helper for some hours, started well and guessing the dates and was in finding albums status a long time. Then the cmd windows closes and there is nothing but an empty ALL_PHOTOS folder in the output path. Tried several times with the same outcome.
Any idea what went wrong?

**Comment by TheriomGit** [comment]
The exact same happened to me. There was also no error message during the process.

**Comment by TheLastGimbus** [comment]
how many gb of photos are you speaking of? what is your ram? 

but generally speaking, this is an issue that few people had, and... i don't know that to do about it yet :( sorry

**Comment by tfernanddes** [comment]
here when it tries to "finding albums..." the app crash and closes without exporting any photo. :( .. any fix?

**Comment by TheLastGimbus** [comment]
merging to #339
